plotly/plotly/plotly.py

```diff
@@ -50,7 +50,7 @@
 DEFAULT_PLOT_OPTIONS = {
     'filename': "plot from API",
     'fileopt': "new",
-    'world_readable': True,
+    'world_readable': False,
```

Comment:
yeah, that sounds better to me! To be clear then, @aneda , let's change this back to `'world_readable': True` to maintain backwards compat.